clarity/contracts/traits/trait-semi-fungible-token.clar

```diff
@@ -0,0 +1,39 @@
+(define-trait semi-fungible-token-trait
```

Comment:
I suggest using the [trait from the SIP](https://github.com/MarvinJanssen/sips/blob/feat/sip-013/sips/sip-013/sip-013-semi-fungible-token-standard.md#trait-definition) and then defining custom features in a separate ALEX-specific trait. This trait does not conform to the current draft which could cause issues down the line. (If it is ratified as-is.)